### DISCUSSION TITLE
Can you have subfolders in page folders?

### DISCUSSION THREAD

**Discussion by Andre601** [mmistakes/minimal-mistakes#3186] [Q&A] [answered]
Right now do I have a structure like this:
```
|- _changelogs/
|- _data/
|- _guides/
|- _includes/
|- _pages/
|- assets/
|- Gemfile
|- _config.yml
```
Each folder prefixed with a `_` contains either MD pages shown on the site or other important files such as theme overrides.

The thing is, that in folders such as `_guides` do I start to have a lot of markdown files, which are all part of the site's `/guides` sub-page.
What is important here is, that some pages belong to other pages in the same folder. As an example is `appearance.md` part of `options.md` and explains some specific settings in more detail.

I would like to order those pages now by having them in separate sub-folders within `_guides` and was wondering, if Minimal Mistakes (Or Jekyll in general) does support having folders within page folders, or if I have to define them too in the `_config.yml`'s include section.

**Comment by mmistakes** [comment, marked as answer]
It depends on your `permalink` structure, but in general Jekyll respects subfolders and adds them to the path.
I would just experiment with it and see if you get the output you want.

I recently [blogged about how Jekyll handles URLs](https://mademistakes.com/mastering-jekyll/how-to-link/) for pages and collections. Looks like you're using collections for `_guides`, so organizing Markdown files beneath that into subfolders should work. It did when I tested it... https://mademistakes.com/mastering-jekyll/how-to-link/#collection-url-defaults

**Reply by Andre601** [reply]
Thanks for the links.
Reading through the pages made me feel safer now because I do have `permalink` values set in the file's frontmatter so the URLs should be unaffected. Thanks for the quick response.

**Reply by mmistakes** [reply]
If that's the case then yeah you're safe. You can structure the files however you want and Jekyll will override the path to match your `permalink`.

As long as the files are in a folder that Jekyll can read, you'll have no problems. And that only ever happens when you use a folder with an `_underscore` and don't add it to the include array inside of `_config.yml`. Unless it's in `_posts`, `_includes`, or `_layouts` as Jekyll reads those by default.
